traffic_ops/app/lib/API/Tenant.pm

```diff
@@ -44,14 +44,16 @@ sub index {
 	my $orderby = $self->param('orderby') || "name";
 	my $rs_data = $self->db->resultset("Tenant")->search( undef, {order_by => 'me.' . $orderby } );
```

Comment:
I assume you mean that I'll go down the tenancy tree instead of testing the permissions tenant by tenant. If I'm correct, you practically assume that the "parent/child" is the only relationship between tenants we have, which may not be the case in the future.